CHANGELOG.md

```diff
@@ -266,48 +273,48 @@ Thanks to the contributions of the community ❤️💙💛💜🧡
 
 * Feature: the `OpenApiSpex` and `OpenApiSpex.Info` structs now support [extensions](https://swagger.io/docs/specification/openapi-extensions/) (#108) (#114)
 
-The `extensions` key may contain any additional data that should be included in the info, eg the `x-logo` and `x-tagGroups` extensions:
-
-```elixir
```

Comment:
Changing the indentation here doesn't feel necessary? It's rendering correctly in github AFAICT

Reply:
To make sure the code example have the same indentation as the bullet list. See screenshots.

With indentation:
![Screenshot-20210502051644-1684x534](https://user-images.githubusercontent.com/134518/116795196-ba142280-ab05-11eb-8787-a8e6e13f8e75.png)

Without indentation (before the change):
![Screenshot-20210502051947-1746x622](https://user-images.githubusercontent.com/134518/116795240-2000aa00-ab06-11eb-984d-5cf7e96e3f90.png)

Reply:
Good catch 👍